readme.txt

```diff
@@ -4,7 +4,7 @@ Tags:              content, distribution, syndication, management
 Requires at least: 5.7
 Tested up to:      6.2
 Requires PHP:      7.4
-Stable tag:        1.9.1
+Stable tag:        2.0.0-beta1
```

Comment:
Stable is intended to point to the [latest](https://github.com/10up/distributor/releases/latest) release (`latest` being defined using the GitHub terminology).

This is not a big deal as Distributor isn't hosted on w.org.